lang/stack/Portfile

```diff
@@ -84,12 +80,12 @@ post-extract {
     xinstall -m 0755 -W ${distpath} ./${distname}-bin ${workpath}/bin
 }
 
-# bootstrap binaries seg fault on 10.7 and older
-if {${os.platform} eq "darwin" && ${os.major} < 12} {
+# bootstrap binaries seg fault on OSX10.7 and older
+if { ${os.major} < 12 } {
     known_fail yes
     pre-fetch {
-        ui_error "${name} @${version} requires OS X 10.8 or later"
-        return -code error "unsupported Mac OS X version"
+        ui_error "${name} is not supported on this macOS release (Darwin ${os.major})."
```

Comment:
Its more user friendly to report the minimum OSX version than the darwin version.
If you want you can compute the OSX10.x minor version from `[expr ${os.major} - 4]`

Reply:
Fixed now